engine/function/src/templates/Numeric.ftl

```diff
@@ -2788,6 +2855,55 @@ public class Numeric {
 
     </#if>
 
+
+    /**
+     * Compares two specified values.  Deephaven null values are less than normal numbers which are less than NaN values.
+     *
+     * @param v1 the first value to compare.
+     * @param v2 the second value to compare.
+     * @returns the value 0 if v1 is numerically equal to v2; a value of less than 0 if v1 is numerically less than v2; and a value greater than 0 if v1 is numerically greater than v2.
+     *      Deephaven null values are less than normal numbers which are less than NaN values. 
```

Comment:
We *might* want to add a line to assuage any IEEE 754 hardliners among our customers. Something like:

Unlike standard Java, we treat NaN values as ordered. In particular two NaN values will compare equal to each other, and a NaN value will compare greater than any other value.